common-util/src/main/java/org/sunbird/common/request/UserRequestValidator.java

```diff
@@ -151,6 +151,22 @@ public static void createUserBasicValidation(Request userRequest) {
               ResponseCode.dataTypeError.getErrorMessage(), JsonKey.LANGUAGE, JsonKey.LIST),
           ERROR_CODE);
     }
+  }
+
+  private void createUserBasicProfileFieldsValidation(Request userRequest) {
+    if (StringUtils.isBlank((String) userRequest.getRequest().get(JsonKey.FIRST_NAME))) {
```

Comment:
BaseRequestValidator.validateParam can be used with mandatoryParamsMissing error.